app/src/main/res/values/strings.xml

```diff
@@ -165,6 +165,11 @@
     <string name="your_time_can_not_be_later_than_time_out_company">Your time can not be later than time out company</string>
     <string name="time_must_be_is_in_a_day">Time must be is in a day</string>
     <string name="request_time_to_can_not_greater_than_request_time_from">Request time to can not greater than request time from</string>
+    <string name="you_have_to_choose_the_check_in_time_first">  You have to choose the check in time first</string>
+    <string name="you_have_to_choose_the_check_out_time_first">  You have to choose the check out time first</string>
```

Comment:
FATAL: [MissingTranslation] "`you_have_to_choose_the_check_out_time_first`" is not translated in "ja" (Japanese), "vi" (Vietnamese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)